br/pkg/restore/split/split_client.go

```diff
@@ -1,11 +1,12 @@
 // Copyright 2020 PingCAP, Inc. Licensed under Apache-2.0.
 
-package restore
+package split
```

Comment:
The file permission shouldn't be changed.

Reply:
It seems the changed file accessing priviledge is ok(644), it original 755 seems has problem.